README.md

```diff
@@ -2,15 +2,9 @@
 
 [![Documentation Status](https://readthedocs.org/projects/microsoft-recommenders/badge/?version=latest)](https://microsoft-recommenders.readthedocs.io/en/latest/?badge=latest)
 
-## What's New (April, 2023)
+## What's New (August, 2023)
 
-We reached 15,000 stars!!
-
-Our latest release is [Recommenders 1.1.1](https://github.com/microsoft/recommenders/releases/tag/1.1.1)! 
-
-We have introduced a new way of testing our repository using [AzureML](https://azure.microsoft.com/en-us/services/machine-learning/). With AzureML we are able to distribute our tests to different machines and run them in parallel. This allows us to test our repository on a wider range of machines and provides us with a much faster test cycle. Our total computation time went from around 9h to 35min, and we were able to reduce the costs by half. See more details [here](tests/README.md).
-
-We also made other improvements like faster evaluation metrics and improving SAR algorithm. 
+We moved to a new organization! Now to access the repo, instead of going to https://github.com/microsoft/recommenders, you need to go to https://github.com/recommenders-team/recommenders. The old URL will still resolve to the new one, but we recommend you to update your bookmarks.
```

Comment:
Maybe it is good to share which org we're joining?
FYI, in [feathr](https://github.com/feathr-ai/feathr) readme:

> It is currently a project under [LF AI & Data Foundation](https://lfaidata.foundation/blog/2022/09/12/feathr-joins-lf-ai-data-as-new-sandbox-project/).
> Read our announcement on Open Sourcing Feathr and Feathr on Azure, as well as the announcement from LF AI & Data Foundation.

Reply:
I haven´t added any notice because I'm waiting for the official comments by both orgs.